src/materials/ConcreteExpansionMicrocrackingDamage.C

```diff
@@ -0,0 +1,167 @@
+/****************************************************************/
+/*               DO NOT MODIFY THIS HEADER                      */
+/*                       BlackBear                              */
+/*                                                              */
+/*           (c) 2017 Battelle Energy Alliance, LLC             */
+/*                   ALL RIGHTS RESERVED                        */
+/*                                                              */
+/*          Prepared by Battelle Energy Alliance, LLC           */
+/*            Under Contract No. DE-AC07-05ID14517              */
+/*            With the U. S. Department of Energy               */
+/*                                                              */
+/*            See COPYRIGHT for full restrictions               */
+/****************************************************************/
+
+#include "ConcreteExpansionMicrocrackingDamage.h"
+#include "ElasticityTensorTools.h"
+
+registerMooseObject("BlackBearApp", ConcreteExpansionMicrocrackingDamage);
+
+template <>
+InputParameters
+validParams<ConcreteExpansionMicrocrackingDamage>()
+{
+  InputParameters params = validParams<ScalarDamageBase>();
+  params.addClassDescription("Scalar damage model based on extent of internal expansion");
+
+  params.addRequiredParam<MaterialPropertyName>(
+      "eigenstrain_name", "name of the eigenstrain driving the microcracking damage process");
```

Comment:
The descriptions of all of these parameters should start out with a capitalized first word for consistency.

Also, I have several parameter renaming suggestions that would probably be good to discuss by phone. How about 'microcracking_eigenstrain_name' for this one (to make it clear which eigenstrain is needed, since there could be multiple)?